### PR TITLE
Update check_modules to add node and check its version as well as rem…

### PR DESCRIPTION
**Pull request by pstaabp**
Update check_modules to add node and check its version as well as remove apache1 modules

### CODE REVIEW

File: bin/check_modules.pl [hunk 1]
```diff
@@ -196,20 +181,30 @@ =head1 DESCRIPTION
 	check_modules(@modulesList);
 }
 
+use Data::Dumper;
```
Comment (drgrice1) [inline bin/check_modules.pl:184]: I think you forgot this!

**Review by drgrice1 — APPROVED** [review]
Looks good.

File: bin/check_modules.pl (hunk 1 above)
Comment (pstaabp) [reply bin/check_modules.pl]: removed it!

Comment (drgrice1) [reply bin/check_modules.pl]: Thanks.

**Comment by drgrice1** [comment]
I will go ahead and merge this.

**Pull request merged**